pkgs/desktops/deepin/apps/deepin-camera/default.nix

```diff
@@ -0,0 +1,94 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, qttools
+, wrapQtAppsHook
+, dtkwidget
+, qt5integration
+, qt5platform-plugins
+, image-editor
+, qtbase
+, qtmultimedia
+, ffmpeg
+, ffmpegthumbnailer
+, libusb1
+, portaudio
+, libv4l
+, gst_all_1
+, systemd
+}:
+
+stdenv.mkDerivation rec {
+  pname = "deepin-camera";
+  version = "1.4.8";
+
+  src = fetchFromGitHub {
+    owner = "linuxdeepin";
+    repo = pname;
+    rev = version;
+    sha256 = "sha256-p2RCetx1lgLonXZaC3umE+nDgZnp64o3iR2MgQhbisM=";
+  };
+
+  patches = [ ./let_libPath_read_LD_LIBRARY_PATH.diff ];
```

Comment:
Ideally we should reuse the functionalities of the dynamic linker, e.g. `dlopen`. Can you provide more context about the use of `libPath` (like its call site) to see if a better solution is viable.

Reply:
1. Used for QLibrary,  but I don't think it's necessary，QLibrary can handle more cases than libPath 

https://github.com/linuxdeepin/deepin-camera/blob/3d3d13818a750964f2d871146f466f21d82fde85/src/src/gstvideowriter.cpp#L75

2. Get the absolute path of the dynamic library，
https://github.com/linuxdeepin/deepin-camera/blob/3d3d13818a750964f2d871146f466f21d82fde85/src/src/mainwindow.cpp#L839-L862
https://github.com/linuxdeepin/deepin-camera/blob/3d3d13818a750964f2d871146f466f21d82fde85/src/src/basepub/load_libs.c#L603

Reply:
So can't we just keep the lib names as is in `CMainWindow::initDynamicLibPath()`, without resolving to absolute paths?

Reply:
Seems good enough for `dlopen` and 'QLibrary'